### PR TITLE
[Re: #157] Use UTC on observations

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

File: network/static/js/observation_view.js [hunk 1]
```diff
@@ -81,5 +81,17 @@ $(document).ready(function() {
         });
     });
 
+    // Delete confirmation
```
Comment (jinxcat) [inline network/static/js/observation_view.js]: How is this connected to the change to UTC?

File: network/static/js/observation_view.js [hunk 2]
```diff
@@ -81,5 +81,17 @@ $(document).ready(function() {
...
+    var message = "Do you really want to delete this Observation?";
+        var actions = $('#obs-delete');
```
Comment (jinxcat) [inline network/static/js/observation_view.js]: typo?

**Comment by jinxcat** [comment]
Are (most of) the changes in network/templates/base/observation_new.html due to indentation?

File: network/static/js/observation_view.js (hunk 1 above)
Comment (comzeradd) [inline network/static/js/observation_view.js]: It's not, but it's not new code either. This was inline javascript on the template and I just moved it to the script file. I'll add a comment on the commit.

**Comment by comzeradd** [comment]
> Are (most of) the changes in network/templates/base/observation_new.html due to indentation?

Yeap. We had the indentation wrong here. The only real change is the addition of the notice under the heading and the new utc script (both similar identical to observation_view template).

**Comment by jinxcat** [comment]
LGTM

**Pull request merged**